views/components/icons.njk

```diff
@@ -104,3 +104,15 @@
         <path d="M94.72 243.18c112.46 0 173.96-93.168 173.96-173.96 0-2.646-.054-5.28-.173-7.903 11.938-8.63 22.314-19.4 30.498-31.66-10.955 4.87-22.744 8.147-35.11 9.625 12.622-7.57 22.313-19.543 26.885-33.817-11.813 7.004-24.895 12.093-38.824 14.84C240.8 8.423 224.916.99 207.326.99c-33.763 0-61.143 27.38-61.143 61.13 0 4.8.537 9.466 1.586 13.942-50.816-2.556-95.875-26.886-126.03-63.88-5.252 9.036-8.28 19.53-8.28 30.73 0 21.212 10.794 39.938 27.208 50.893-10.03-.31-19.454-3.063-27.69-7.646-.01.255-.01.505-.01.78 0 29.61 21.076 54.33 49.052 59.933-5.137 1.4-10.543 2.152-16.122 2.152-3.933 0-7.766-.387-11.49-1.103 7.783 24.293 30.354 41.97 57.114 42.465-20.926 16.402-47.287 26.17-75.937 26.17-4.93 0-9.798-.28-14.584-.845 27.06 17.344 59.19 27.464 93.723 27.464"></path>
     </svg>
 {% endmacro %}
+
+{% macro iconTick() %}
+    <svg viewbox="0 0 100 96"
+        width="20"
+        height="20"
+        xmlns="http://www.w3.org/2000/svg"
+    >
+        <title>tick by mikicon from the Noun Project</title>
```

Comment:
Sorry to ⛏ but this title could do with updating.

Reply:
I got the icon from the Noun Project which requires some attribution, couldn't think how else to do it..?

Reply:
Fair enough, go for it then.